tools/cockpit.spec

```diff
@@ -52,6 +52,10 @@ BuildRequires: docbook-style-xsl
 BuildRequires: keyutils-libs-devel
 BuildRequires: dbus-devel
 BuildRequires: glib-networking
+
+# There is a segfault in g_source_add_child_source that was fixed in
+# 2.37.3-2-g1d5c815
+BuildRequires: glib2-devel >= 2.40
```

Comment:
Also, it appears that I have 2.40 in one place and 2.37.3 in another. I started with 2.40 then narrowed it down, but forgot to change both locations...